docs/guides/basic-rbac.mdx

```diff
@@ -47,48 +47,53 @@ declare global {
 
 ## Set the admin role for your user
 
-Later, you will add a basic admin tool to change the user's role, but for now, let's manually add a role to your own user account. In the Clerk Dashboard, navigate to [**Users**](https://dashboard.clerk.com/last-active?path=users) and select your own user account. Scroll down to the **Metadata** section and next to the **Public** option, select **Edit**. Add the following JSON and select **Save**.
+Later in the, you will add a basic admin tool to change a user's role. For now, let's manually add a role to your own user account.
+
+1. In the Clerk Dashboard, navigate to [**Users**](https://dashboard.clerk.com/last-active?path=users) and select your own user account.
+1. Scroll down to the **User metadata** section and next to the **Public** option, select **Edit**.
+1. Add the following JSON and select **Save**.
 
 ```json
 {
   "role": "admin"
 }
 ```
 
-![The Users page in the Clerk Dashboard with the 'Edit public metadata' modal open. The modal has a text field with the JSON 'role': 'admin'.](/docs/images/guides/basic-rbac/edit-public-metadata.webp)
+## Create and protect an admin dashboard
 
-## Create an admin dashboard and protect it
+Now that your user has the `admin` role, let's build an admin dashboard manage user roles easily.
 
-Now that your user has the `admin` role, let's build an admin dashboard to help you easily change users' roles. In your `app/` directory, create an `admin/` folder. Within the `admin/` folder, create a `dashboard/` folder. In the `dashboard/` folder, create a file named `page.tsx`. Copy the following code and paste it into the file.
+1. In your `app/` directory, create an `admin/` folder.
+1. Inside the `admin/` folder, create a `dashboard/` folder
```

Comment:
```suggestion
```

The `/admin/dashboard` double layer is confusing. I don't recall how that got into the original guide, but there its not reflective of a feature like this. In the below suggestions I am changing it to just `/admin` though just `/dashboard` would also work.